src/musubi_tuner/gui/gui.py

```diff
@@ -824,7 +825,7 @@ def start_training(
                 "--network_dim",
                 str(int(dim)),
                 "--optimizer_type",
-                "adamw8bit",
+                "adamw" if platform.system() == "Darwin" else "adamw8bit",
```

Comment:
I think really the condition should be "If running on Mac OS or otherwise device is CPU" but not sure of a clean way to do this here.

Reply:
yea that's fine. leave it for now until more people actually use it i guess